### ISSUE TITLE
User feedback: "Type" facet for different book type and chapter type attributes is misleading

### ISSUE THREAD

**Issue by ssmulyan** [CrossRef/cr-search#53]
For example, in searching for the book "Signal Processing of Power Quality Disturbances" in CRMDS, the correct result doesn't appear when you search for that title.

https://search.crossref.org/?q=Signal+Processing+of+Power+Quality+Disturbances

But, when you limit the results to just "Book" type, the item also doesn't appear in results
https://search.crossref.org/?q=Signal+Processing+of+Power+Quality+Disturbances&type-name=Book

That's because it's classed as a Monograph, rather than Book, due to the <book book_type="monograph"> attribute in the deposited XML

No Monograph facet appears in the CRMDS interface when a user searches for that book title, but if you added manually, the correct item appears as the first result
https://search.crossref.org/?q=Signal+Processing+of+Power+Quality+Disturbances&type-name=Monograph

DOI for this example
10.1002/0471931314

Most recent metadata deposit
https://doi.crossref.org/servlet/submissionAdmin?sf=content&submissionID=318055038
including book_type=monograph

in REST API
http://api.crossref.org/works/10.1002/0471931314
"type": "monograph",

TL;DR
An item being a "monograph" (or "edited book" or "reference work") doesn't make it "not a book".  Those are just sub-types of books.  So, if we're only going to be displaying the "Book" filter/facet, it should encompass all the book types.
